lib/src/analyzers/lint_analyzer/rules/rules_list/format_comment/visitor.dart

```diff
@@ -11,6 +11,10 @@ const _ignoreExp = 'ignore:';
 const _ignoreForFileExp = 'ignore_for_file:';
 
 class _Visitor extends RecursiveAstVisitor<void> {
+  _Visitor(this._ignoredPatterns);
+
+  final Iterable<RegExp> _ignoredPatterns;
```

Comment:
And please reorder variables as here https://github.com/dart-code-checker/dart-code-metrics/blob/master/lib/src/analyzers/lint_analyzer/rules/rules_list/avoid_returning_widgets/visitor.dart#L15

Reply:
I moved the variables in this commit: https://github.com/dart-code-checker/dart-code-metrics/pull/828/commits/5ef888526b251c5dbacaf2e7c2689ab933ed06ad